nowcasting_dataset/geospatial.py

```diff
@@ -120,3 +122,46 @@ def calculate_azimuth_and_elevation_angle(
 
     # extract the information we want
     return solpos[["elevation", "azimuth"]]
+
+
+def get_osgb_center_from_osgb(
+    x_osgb: Union[xr.DataArray, List[float], np.ndarray],
+    y_osgb: Union[xr.DataArray, List[float], np.ndarray],
+) -> (float, float):
+    """
+    Get the center OSGB coords from a list of OSGB coords.
+
+    Args:
+        x_osgb: list of x coords in OSGB
+        y_osgb: list of y coords in OSGB
+
+    Returns: x and y center [OSGB]
+
+    """
+    if type(x_osgb) == xr.DataArray:
```

Comment:
I don't think it's necessary to switch on the type of the object because I think `np.mean(x_osgb)` will do the right thing, no matter if `x_osgb` is a `np.ndarray`, an `xr.DataArray`, or a `list`.   Here's a quick experiment I did:

![image](https://user-images.githubusercontent.com/460756/141987582-24d60c4c-eb31-4be1-be49-42a57cc2a726.png)

Reply:
Thanks, not sure why i didnt keep it simple - thanks